src/pages/our-work.js

```diff
@@ -41,18 +30,42 @@ const ourWork = {
   seoTitle: 'A collection of case studies'
 }
 
+const IntroTitleCol = styled(Col)`
+  padding-top: ${({ theme }) => theme.space[5]};
+
+  ${breakpoint('tablet')`
+    padding-top: ${({ theme }) => theme.space[6]};
+  `}
+`
+
+const IntroDescriptionCol = styled(Col)`
+  padding-bottom: ${({ theme }) => theme.space[5]};
+
+  ${breakpoint('tablet')`
+    padding-bottom: ${({ theme }) => theme.space[7]};
+  `}
+`
+
 const OurWork = ({ data }) => {
   const {
+    allContentfulNonTemplatedCaseStudyV2,
     allContentfulTemplatedCaseStudy,
     allContentfulNonTemplatedCaseStudy,
-    allContentfulNonTemplatedCaseStudyV2
+    contentfulOurWork: { caseStudies }
   } = data
 
+  const displayOrderByIDs = caseStudies
+    .filter(({ publish }) => publish)
+    .map(({ id }) => id)
+
+  const getCSOrderIndex = id =>
+    displayOrderByIDs.indexOf(displayOrderByIDs.find(cs => id.includes(cs)))
```

Comment:
You don't need the find in this instance. If indexOf can't find the item in the array it will return `-1`, if the find method can't find it it just returns undefined so if the id is not in the array you're doing `ids.indexOf(undefined)` if you include the `.find` here.

Reply:
Wow! So true. How come I left this?? Updated. Now gonna have a look at your fiddle!